roles/certbot/files/rsync.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Start the rsync server and perform the transfer
+rrsync -wo /etc/letsencrypt/live
```

Comment:
I don't fully understand this. Won't this hang the certbot hook forever? Do we not want this to push to the hosts instead of waiting them to pull?

Reply:
like, I understand that rsync is started below in the hook - but do we actually have a guarantee that they will all be ran at the same time? If not, do we need the rsync server or can we just rely on rsync to start it over e.g. `-e ssh`?

Reply:
yea, we likely don't need the `rsync.sh` script at all. Not enreily sure what the nginx restart is for, but if it's needed, we could move it to a handler

Reply:
https://github.com/python-discord/infra/pull/105/commits/8248baf78f82bee45958583f570104506f82face